optuna/samplers/tpe/sampler.py

```diff
@@ -86,6 +86,38 @@ def objective(trial):
             study = optuna.create_study(sampler=TPESampler())
             study.optimize(objective, n_trials=10)
 
+    Args:
+        consider_prior:
+            Enhance the stability of Parzen estimator by imposing a Gaussian prior when True. The
```

Comment:
Suggestion to align to the Optuna doc convention:
```suggestion
            Enhance the stability of Parzen estimator by imposing a Gaussian prior when :obj:`True`. The
```